ViSearchSDK/ViSearchAPISource/BaseSearchParams.m

```diff
@@ -44,14 +46,7 @@ - (NSDictionary *)toDict{
         [dict setValue:facet? @"true":@"false" forKey:@"facet"];
         [dict setValue:[NSString stringWithFormat:@"%d", facetSize] forKey:@"facet_size"];
         if (facetField!= nil) {
-            NSMutableString* builder = [@"" mutableCopy];
-            
-            for (int i=0; i<[facetField count]; i++) {
-                [builder appendString:[facetField objectAtIndex: i]];
-                if(i < [facetField count]-1)
-                    [builder appendString:@","];
-            }
-            [dict setValue:builder forKey:@"facet_field"];
+            [dict setValue:facetField forKey:@"facet_field"];
```

Comment:
@yulu dictionary can only contain one value for a key, so it will only keep the last added facet_fields.